homework.md

```diff
@@ -4,77 +4,77 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12,240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language 
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+It is the styling of the website i.e. the colours, the fonts etc. 
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
+h1 {
+    whatever command you would like. 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+It is a text-based programming language. 
```

Comment:
Hi Monique! Well done! You could include a bit more detail about JavaScript here. Like the fact that it is generally used for web-pages or some other similar fact.